scripts/mixins/families/euvhi.lua

```diff
@@ -46,13 +41,9 @@ local function closeFlower(mob)
     mob:setMod(xi.mod.SLASH_SDT, 1000)
     mob:setMod(xi.mod.PIERCE_SDT, 1000)
     mob:setMod(xi.mod.IMPACT_SDT, 1000)
-    for n =1, #xi.magic.resistMod, 1 do
-        mob:setMod(xi.magic.resistMod[n], 0)
-    end
-    for n =1, #xi.magic.defenseMod, 1 do
-        mob:setMod(xi.magic.defenseMod[n], 0)
-    end
-    mob:setLocalVar("[euvhi]changeTime", mob:getBattleTime() + math.random(45, 60))
+    mob:setMod(xi.mod.UDMG, 0) -- Takes predicted damage when open
+
+    mob:setLocalVar("[euvhi]changeTime", mob:getBattleTime() + 80) -- Flower will open after 80 seconds
```

Comment:
Duration is being increased here to a flat static number. Has that been checked against retail?